app/uk/gov/hmrc/gform/playcomponents/GformSessionCookieCryptoFilter.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.playcomponents
+
+import _root_.akka.stream.Materializer
+import cats.syntax.eq._
+import com.typesafe.config.Config
+import play.api.mvc._
+import play.api.routing.Router.RequestImplicits._
+import uk.gov.hmrc.crypto.{ Crypted, Decrypter, Encrypter, PlainText }
+import uk.gov.hmrc.gform.FormTemplateKey
+import uk.gov.hmrc.gform.controllers.CookieNames._
+import uk.gov.hmrc.gform.gformbackend.GformConnector
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ Anonymous, FormTemplate, FormTemplateId }
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.HeaderCarrierConverter
+import uk.gov.hmrc.play.bootstrap.frontend.filters.crypto.{ SessionCookieCrypto, SessionCookieCryptoFilter }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class GformSessionCookieCryptoFilter(
+  sessionCookieCrypto: SessionCookieCrypto,
+  nonanonymouscookieCryptoFilter: SessionCookieCryptoFilter,
```

Comment:
Why is cookie here in lower case, whereas elsewhere it is typed with upper case?

Also rather than calling it `nonanonymouscookieCryptoFilter`  should better be to call it:
```
mdtpCookieCryptoFilter
```
or 
```
hmrcCookieCryptoFilter
```

Reply:
Changed to **hmrcCookieCryptoFilter**. In fact, rename all nonanonymous*** names to hmrc***